resources/eventing/charts/nats/README.md

```diff
@@ -23,6 +23,14 @@ $ kubectl -n nats port-forward nats-1 4222
 
 ## Configuration
 
+### Server Image
+
+```yaml
+nats:
+  image: nats:2.8.4-alpine
+  pullPolicy: IfNotPresent
+```
```

Comment:
Lets not keep this `Server Image` information in readme, otherwise we have to maintain it everytime we bump the image.

Reply:
I wanted to be stay in-sync with the original docs as much as possible, but you are also right. I removed it.